examples/static-site-gen/platform/src/lib.rs

```diff
@@ -240,3 +290,157 @@ pub fn strip_windows_prefix(path_buf: PathBuf) -> std::path::PathBuf {
 
     std::path::Path::new(path_str.trim_start_matches(r"\\?\")).to_path_buf()
 }
+
+const FIXED_TOKENS: &[(&str, Token)] = &[
```

Comment:
IMO this part ideally belongs integrated into parse/src/highlight.rs. That said, this is fine for now; it's not really hurting anything.

Reply:
Thank you. I've updated with this change.